integration/test/Test/Demo.hs

```diff
@@ -174,22 +175,10 @@ testIndependentESIndices = do
 
 testDynamicBackendsFederation :: HasCallStack => App ()
 testDynamicBackendsFederation = do
-  startDynamicBackends [def, def] $ \dynDomains -> do
-    [aDynDomain, anotherDynDomain] <- pure dynDomains
-    _ <- Internal.createFedConn anotherDynDomain (Internal.FedConn aDynDomain "full_search")
-    threadDelay 2_000_000
-
-    u1 <- randomUser aDynDomain def
-    u2 <- randomUser anotherDynDomain def
-    uid2 <- objId u2
-    Internal.refreshIndex anotherDynDomain
-
-    bindResponse (Public.searchContacts u1 (u2 %. "name") anotherDynDomain) $ \resp -> do
```

Comment:
did you remove this search intentionally?

Reply:
Yes, the test is supposed to show that federation works. Using search here is just making the test more complicated.